{"name": "test_length[params1]", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'NoneType' and 'int'", "trace": "self = <test_cal.test_case.test_add.TestAdd object at 0x0000020248A92FA0>\nparams = {'data': {'a': 1, 'b': None}, 'name': 'b为空', 'validate': '报错啦'}\n\n    @pytest.mark.add\n    @pytest.mark.p1\n    @pytest.mark.parametrize(\"params\", read_yaml(\"./data/add_length.yaml\"))\n    def test_length(self, params):\n        \"\"\"\n        测试输入参数的长度\n        :param params: a,b,result\n        :return:\n        \"\"\"\n        cal = Calculator()\n>       result = cal.add(params[\"data\"]['a'], params[\"data\"]['b'])\n\ntest_case\\test_add.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <calculator.Calculator object at 0x0000020248A9E3D0>, a = 1, b = None\n\n    def add(self, a, b):\n>       if a > 99 or a < -99 or b > 99 or b < -99:\nE       TypeError: '>' not supported between instances of 'NoneType' and 'int'\n\n..\\calculator.py:8: TypeError"}, "description": "\n        测试输入参数的长度\n        :param params: a,b,result\n        :return:\n        ", "parameters": [{"name": "params", "value": "{'name': 'b为空', 'data': {'a': 1, 'b': None}, 'validate': '报错啦'}"}], "start": 1649942265779, "stop": 1649942265779, "uuid": "d94c4042-81b1-493d-90d6-a3b43be255a6", "historyId": "8ba6672475b6fda9830b8f5e0466f8c1", "testCaseId": "18fc6f6f5dac2aec77cfa2769d0346e9", "fullName": "test_case.test_add.TestAdd#test_length", "labels": [{"name": "tag", "value": "p1"}, {"name": "tag", "value": "add"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "Daisy"}, {"name": "thread", "value": "3016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_add"}]}